invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm_requests/menuTabInit.js

```diff
@@ -0,0 +1,22 @@
+import $ from "jquery";
+
+const tabElementSelector = "#request-community-submission-tab .item";
+const tabElement = $(tabElementSelector);
+
+tabElement.tab({
+  onVisible: function (tab) {
+    $(tabElementSelector).attr("aria-selected", false);
+    $(`#${tab}-tab-menu`).attr("aria-selected", true);
+
+    $(` .tab.segment`).attr("hidden", true);
```

Comment:
```suggestion
    $('.tab.segment').attr("hidden", true);
```
it this line correct? It will look for all the `.tab.segment` elements.

Suggestion: to make it more efficient, it would be nice to wrap all `.tab.segment` inside an `id` so that we can do:
```
$('#myid').find('.tab.segment').attr("hidden", true);
```